### PR TITLE
Mount server separate binary

### PR DESCRIPTION
**Pull request by smalyala**
- Separate mount server into its own binary
- Build and publish docker image via CI

### CODE REVIEW

File: src/internal/minikubetestenv/deploy.go [hunk 1]
```diff
@@ -478,6 +478,9 @@ func putRelease(t testing.TB, ctx context.Context, namespace string, kubeClient
 	if opts.EnterpriseMember {
 		helmOpts = union(helmOpts, withEnterpriseMember(pachAddress.Host, int(pachAddress.Port)))
 	}
+	if opts.EnterpriseMember {
```
Comment (chainlink) [inline src/internal/minikubetestenv/deploy.go:481]: This looks unrelated?

**Comment by chainlink** [comment]
Heads up @molinamelendezj This adds a docker build to goreleaser.

File: src/internal/minikubetestenv/deploy.go (hunk 1 above)
Comment (smalyala) [reply src/internal/minikubetestenv/deploy.go]: Good catch. Removed

**Review by chainlink — APPROVED** [review]
